stories/2022-03-27_Glass_ML/ML_20th_Century.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>Jennifer Ding:</p><ul><li>Could you explain the data columns earlier? I gather that most of them are element measurements, but it might help clarify the data to have a data dictionary that maps the column name to semantic meaning (e.g. Refractive index -&gt; 'Ref_ix' -&gt; "Refractive index refers to...")</li></ul><p>Joe Cerniglia:</p><p>I agree there is a real need here for explanations of the columns in the glass index.&nbsp;I can provide explanations.&nbsp;However, might it be better to place this directly in the story text, rather than within a data dictionary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>